src/ResourceManager/Kusto/Commands.Kusto/Commands/GetAzureRmKustoCluster .cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
There is a space at the end of the filename. Please remove the space, `GetAzureRmKustoCluster .cs`